src/express/commands/shadow.js

```diff
@@ -0,0 +1,81 @@
+import Web3 from 'web3'
+import { loadDevnetConfig, splitToArray } from '../common/config-utils'
+import { maxRetries, runSshCommand } from '../common/remote-worker'
+import { stopServices } from './cleanup'
+
+async function initWeb3(provider) {
+  return new Web3(provider)
+}
+
+export async function shadow(targetBlock) {
+  require('dotenv').config({ path: `${process.cwd()}/.env` })
```

Comment:
Can we validate `targetBlock ` before anything else?
We do `latestBlock.number > targetBlock` later on but it's worth to check that `targetBlock` is a valid param (number, no null, no undefined, no empty string)

Reply:
Done: [88a5694](https://github.com/maticnetwork/matic-cli/pull/157/commits/88a56943bc7a38b12a738340a89a6db70e4bd69f)